src/app/components/Footer.tsx

```diff
@@ -0,0 +1,72 @@
+"use client"
+import { Header, PrimaryNav } from "@trussworks/react-uswds";
+import Link from "next/link";
+import Image from "next/image";
+import React from "react";
+import styles from "../styles/Home.module.scss"
+import { NavigationLink} from "./Header"
+
+export default function Footer() {
+    const basePath = "/dibbs-site";
+
+    const [expanded, setExpanded] = React.useState(false)
+    const onClick = () => { if (window.innerWidth < 1024) setExpanded((prvExpanded) => !prvExpanded) }
+
+    const testItemsMenu = [
+        <Link
+            href='/'
+            key="one"
+            className={`usa-nav__link ${styles.homeNavItem}`}
+            onClick={onClick}>
+
+            <span className={styles.navbarItemText}>Home</span>
+
+        </Link>,
+        (
+          <NavigationLink key="two" href={`${basePath}/`} onClick={onClick}>
```

Comment:
We should be able to get rid of the `basePath` in these URLs. I just noticed the footer's links are doubling up on it. For example, when I click on `Our products` I get a URL like this: `http://localhost:3000/dibbs-site/dibbs-site/our-products`

The links in the header seem to be correct and work fine 👍